dolly-backend-app/src/main/java/no/nav/dolly/bestilling/service/DollyBestillingService.java

```diff
@@ -193,7 +193,7 @@ public void oppdaterPersonAsync(RsDollyUpdateRequest request, Bestilling bestill
                         oppdaterPersonResponse.getIdentTupler().stream()
                                 .map(RsOppdaterPersonResponse.IdentTuple::getIdent).collect(toList()), null, progress);
 
-                dollyPerson.set(dollyPersonCache.prepareTpsPersoner(oppdaterPersonResponse));
+                dollyPerson.set(dollyPersonCache.prepareTpsPersoner(oppdaterPersonResponse, bestilling.getIdent()));
```

Comment:
Med feilen som er blitt konstantert (som skyldes en endring i TPSF, men som gjør mekanismen ekstra sårbar for endringer) så er det bedre å starte på ny med bare ident på hovedperson. Det koster et kall ekstra til TPSF, men gjør koden mer robost for bedre vedlikehold.
Tenker at metode prepareTpsPersoner(RsOppdaterPersonResponse identer)  i DollyPersonCache kan slettes og erstattes av
```
public DollyPerson prepareTpsPerson(String ident) {
      return DollyPerson. builder()
           .hovedperson(ident)
           .master(Master.TPSF)
           .build();
} 
```
eventuelt å ha denne kode direkte i DollyBestillingService

Reply:
Åja, det høres ut som en god ide, skal fikse det 😄